### ISSUE TITLE
Flow view zoom with Ctrl + mousewheel too sensitive

### ISSUE THREAD

**Issue by askor** [a85/POSTMan-Chrome-Extension#12920]
### Is there an existing issue for this?

- [X] I have searched the tracker for existing similar issues and I know that duplicates will be closed

### Describe the Issue

When using `Ctrl + mousewheel` to zoom when viewing a flow, one click on my mouse takes me either to maximum zoom out, or all the way in. The granularity is not the same as the zoom buttons in the bottom toolbar.

If this is intended, then it works different from all other apps with similar views. `Ctrl + scroll` should have sufficient granularity to zoom to your preferred level.

### Steps To Reproduce

1. Create a new flow
2. Add an element, like a request (to see sizes)
3. Hold `Ctrl `and scroll back one notch on the mousewheel
4. Still holding `Ctrl`, scroll up on notch
5. You are now at maximum zoom in

### Step 2 
![image](https://github.com/postmanlabs/postman-app-support/assets/11080290/2c4f0eaa-847d-45f8-b376-90f927f8c8dd)
### Step 3
![image](https://github.com/postmanlabs/postman-app-support/assets/11080290/573026c0-cae5-46d6-9fd0-10bc65f4713c)
### Step 4
![image](https://github.com/postmanlabs/postman-app-support/assets/11080290/15b499e8-b41b-4249-b9fb-cdf46d95e5e1)

### Operating System

Windows

### Postman Version

11.1.25

### Postman Platform

Postman App

### User Account Type

Signed In User

**Comment by t-shotgnu** [comment]
Absolutely, I thought I have too high mouse zoom sensitivity set on but after I came home from work (in other words, checked on other device), found out that it is an app issue. This thread came across as a first result in search thus it is sick that this was not addressed since the report.

And yeah, this indeed is very annoying to use the zoom in lower left corner.
